build.gradle.kts

```diff
@@ -3,7 +3,7 @@ import org.hypertrace.gradle.publishing.License
 
 plugins {
   id("org.hypertrace.ci-utils-plugin") version "0.2.0"
-  id("org.hypertrace.publish-plugin") version "0.3.3" apply false
+  id("org.hypertrace.publish-plugin") version "0.4.4-SNAPSHOT" apply false
```

Comment:
The snapshot version will be removed once the new `publish-plugin` version is published to artifactory.

Reply:
Is this one you manually pushed for bootstrapping?

Reply:
yes, this one is pushed manually to publish the artifacts from this repo.

Reply:
So the snyk scan failing indicates to me that this is going to fail to publish - I'm not sure why that fails and the build passes though. As another data point, it fails with the same issue when I try to build this branch locally.

Reply:
fixing the synk scan. I've removed the SNAPSHOT version from artifactory for testing purpose.

Reply:
@aaron-steinfeld synk scan has passed. please try building now.